tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -868,38 +868,6 @@ def test_aggregation_alias_comparison(self):
         data["start_timestamp"] = iso_format(before_now(minutes=1, seconds=3))
         event = self.store_event(data, project_id=project.id)
 
-        with self.feature("organizations:discover-basic"):
```

Comment:
I folded two tests together as () less function calls are deprecated and something I'm planning on removing very soon.